libs/common/src/lib/user/onboarding/onboarding.models.ts

```diff
@@ -1,8 +1,8 @@
 export interface OnboardingPayload {
-  company: string;
-  industry: string;
-  job_title: string;
+  company?: string;
+  industry?: string;
+  job_title?: string;
   phone?: string;
-  other_search_engines: string;
-  receive_updates: boolean;
+  other_search_engines?: string;
+  receive_updates?: boolean;
```

Comment:
why setting all the fields as optional? As far as I remember all the fields in the form are actually mandatory. Job title is sent later on, but I feel like that should not impact the other fields.

Reply:
Carles made all the fields optional, that we can call it a first time with all the fields and a second time with job_title only.
We could have imagined a more precise typing but in general I prefer to align with the backend endpoint typing.

Reply:
ok